libs/stats/tests/test_utils.py

```diff
@@ -117,3 +124,180 @@ def test_bin_seasonal_mk_season_rules(months, anchor):
+        'dtr:end_datetime': '2020-04-04T23:33:24.461679Z',
+        'eo:sun_elevation': 32.7056476,
+        'landsat:wrs_path': 91,
+        'dtr:start_datetime': '2020-04-04T23:32:56.662365Z',
+        'odc:product_family': 'wo',
+        'odc:dataset_version': '1.6.0',
+        'dea:dataset_maturity': 'final',
+        'odc:collection_number': 3,
+        'odc:naming_conventions': 'dea_c3',
+        'odc:processing_datetime': '2020-04-04T23:33:10.644420Z',
+        'landsat:landsat_scene_id': 'LE70910862020095ASA00',
+        'landsat:collection_number': 1,
+        'landsat:landsat_product_id': 'LE07_L1TP_091086_20200404_20200501_01_T1',
+        'landsat:collection_category': 'T1'},
+        'measurements': {'water': {'path': 'ga_ls_wo_3_091086_2020-04-04_final_water.tif'}}
+    }
+    
+    fc_metadata = {
+        'id': '41980746-4f17-5e0c-86a0-92cca8d3c99d',
+        'crs': 'epsg:32655',
+        'grids': {'default': {'shape': [7211, 8311], 'transform': [30.0, 0.0, 423285.0, 0.0, -30.0, -4040385.0, 0.0, 0.0, 1.0]}},
+        'label': 'ga_ls_fc_3_091086_2020-04-04_final',
+        '$schema': 'https://schemas.opendatacube.org/dataset',
+        'product': {'name': 'ga_ls_fc_3'},
+        'properties': {'title': 'ga_ls_fc_3_091086_2020-04-04_final',
+        'eo:gsd': 30.0,
+        'created': '2021-03-10T04:14:49.645196Z',
+        'datetime': '2020-04-04T23:33:10.644420Z',
+        'proj:epsg': 32655,
+        'proj:shape': [7211, 8311],
+        'eo:platform': 'landsat-7',
+        'odc:product': 'ga_ls_fc_3',
+        'odc:producer': 'ga.gov.au',
+        'eo:instrument': 'ETM',
+        'eo:cloud_cover': 44.870310145260326,
+        'eo:sun_azimuth': 49.20198554,
+        'proj:transform': [30.0, 0.0, 423285.0, 0.0, -30.0, -4040385.0, 0.0, 0.0, 1.0],
+        'landsat:wrs_row': 86,
+        'odc:file_format': 'GeoTIFF',
+        'odc:region_code': '091086',
+        'dtr:end_datetime': '2020-04-04T23:33:24.461679Z',
+        'eo:sun_elevation': 32.7056476,
+        'landsat:wrs_path': 91,
+        'dtr:start_datetime': '2020-04-04T23:32:56.662365Z',
+        'odc:product_family': 'fc',
+        'odc:dataset_version': '2.5.0',
+        'dea:dataset_maturity': 'final',
+        'odc:collection_number': 3,
+        'odc:naming_conventions': 'dea_c3',
+        'odc:processing_datetime': '2020-04-04T23:33:10.644420Z',
+        'landsat:landsat_scene_id': 'LE70910862020095ASA00',
+        'landsat:collection_number': 1,
+        'landsat:landsat_product_id': 'LE07_L1TP_091086_20200404_20200501_01_T1',
+        'landsat:collection_category': 'T1'},
+        'measurements': {'bs': {'path': 'ga_ls_fc_3_091086_2020-04-04_final_bs.tif'},
+        'pv': {'path': 'ga_ls_fc_3_091086_2020-04-04_final_pv.tif'},
+        'ue': {'path': 'ga_ls_fc_3_091086_2020-04-04_final_ue.tif'},
+        'npv': {'path': 'ga_ls_fc_3_091086_2020-04-04_final_npv.tif'}}
+    }
+    
+    uris = [m["path"] for m in wo_metadata["measurements"].values()]
```

Comment:
These are not uris of measurements, this should be something like `["s3://bucket/fc/003/020/some.yml"]`